examples/diff/index.html

```diff
@@ -0,0 +1,141 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <!-- Core extensions CSS from Contentful. -->
+  <link rel="stylesheet"
+        href="//contentful.github.io/ui-extensions-sdk/cf-widget-api.css">
+  <!-- Styles for added / removed text -->
+  <style>
+    .diff-text .added {
+      font-weight: bold;
+      color: #5cb85c;
+    }
+    .diff-text .removed {
+      text-decoration: line-through;
+      font-weight: bold;
+      color: #950B02;
+    }
+  </style>
+</head>
+<body>
+
+<!-- Extension DOM Elements -->
+<div class="cf-form-field">
+  <input type="text" class="cf-form-input draft-text">
+  <div class="cf-form-hint">
+    <span class="diff-text"></span>
+  </div>
+</div>
+
+<!-- Contenful UI Extensions SDK API -->
+<script src="//contentful.github.io/ui-extensions-sdk/cf-widget-api.js"></script>
+<!-- Diff Lib -->
+<script src="//cdnjs.cloudflare.com/ajax/libs/jsdiff/2.2.2/diff.min.js"></script>
+
+<!-- Diffing UI Extension -->
+<script>
+
+  function DiffExtension(extensionsApi) {
```

Comment:
Capitalization makes this seem like a `class` that should be instantiated using `new`. `diffExtension` is better in my opinion as it's meant to be the callback passed to `init()`.